lib/model/list_model.dart

```diff
@@ -67,6 +72,20 @@ class ListModel {
       await DatabaseManager.updateListModelItems(this);
     }
   }
+
+  // void addLabel(String label) {
```

Comment:
Why are these commented out?

Reply:
I'm sorry, I should have removed them. I ended up not needing them. Will remove.